data-syncs/building-data-syncs/README.md

```diff
@@ -1,161 +1,138 @@
-# Build data syncs
+# Build data syncs in Cinchy
 
-## Guide
+Data syncs in Cinchy synchronizes information between different systems. This guide walks you through the process of setting up your data syncs.
 
-You can use this checklist in conjunction with the documentation below and elsewhere in this space to configure your data syncs in Cinchy.
+## Basic workflow
 
-- [ ] [Install any applicable components](../installation-and-maintenance/)
-- [ ] Determine your [Data Sync Type](types-of-data-syncs.md) and [Design Pattern](common-design-patterns.md)
-- [ ] Define and Configure your [Data Sync Source](../supported-data-sync-sources/)
-- [ ] Define and Configure your [Data Sync Destination](../supported-data-sync-destinations/)
-- [ ] Define and Configure your [Sync Actions](sync-actions.md)
-- [ ] Define and Configure your [Event Stream source](../supported-real-time-sync-stream-sources/) (if running a real-time sync)
+The basic workflow of a data sync in Cinchy is:
 
-## Prerequisites
+1. [Install Necessary Components](link-to-installation-doc).
+1. [Choose Data Sync Type and Design Pattern](link-to-sync-type-pattern).
+1. [Define and Configure Data Sync Source](link-to-configure-source).
+1. [Set Up Data Sync Destination](link-to-setup-destination).
+1. [Specify Sync Actions](link-to-sync-actions).
+1. [Configure Event Stream Source (For Real-time Syncs)](link-to-event-stream).
```

Comment:
This isn't a distinct step. The step `[Define and Configure Data Sync Source]` can either be batch or realtime

Reply:
Removed. It should be explained in [data sync source]